metricbeat/include/list.go

```diff
@@ -8,6 +8,22 @@ package include
 
 import (
 	// This list is automatically generated by `make imports`
+	_ "github.com/elastic/beats/metricbeat/module/system"
```

Comment:
Yes it should be stored alpahbetically. This happened  when i merged the master branch. I will update the code. Many thanks @ruflin for your help!